040-SDK/150-file-attachments.mdx

```diff
@@ -2,30 +2,46 @@
 title: File attachments
 navTitle: File attachments
 keywords: ['files', 'file storage', 'file attachments']
-description: Xata effectively keeps track of files within a database
+description: Store files within your database with Xata
```

Comment:
@snide I changed this so it shows up in the diff. Any better ideas?

Reply:
That's fine.